tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -996,15 +996,15 @@ public function selfClosingTagDataProvider(): array
     {
         return \array_map(
             /**
-             * @param array{0: string, 1: string} $dataset
+             * @param array{0: non-empty-string, 1: non-empty-string} $dataset
              *
-             * @return array{0: string, 1: string, 2:string}
+             * @return array{0: non-empty-string, 1: non-empty-string, 2:non-empty-string}
              */
             static function (array $dataset): array {
                 $updatedDataset = $dataset;
                 \array_unshift($updatedDataset, \str_replace('/>', '>', $dataset[0]));
 
-                /** @var array{0: string, 1: string, 2:string} $updatedDataset */
+                /** @var array{0: non-empty-string, 1: non-empty-string, 2:non-empty-string} $updatedDataset */
```

Comment:
While we're here, we should probably add a space after `2:` for consistency.


```suggestion
                /** @var array{0: non-empty-string, 1: non-empty-string, 2: non-empty-string} $updatedDataset */
```